pkg/controller/keyvault/ingress_secret_provider_class_test.go

```diff
@@ -337,26 +328,26 @@ func TestIngressSecretProviderClassReconcilerBuildSPCInvalidURLs(t *testing.T) {
 		ing.Spec.IngressClassName = nil
 		ing.Annotations = map[string]string{"kubernetes.azure.com/tls-cert-keyvault-uri": "inv@lid URL"}
 
-		ok, err := i.buildSPC(ing, &secv1.SecretProviderClass{})
+		ok, err := BuildSPC(ing, &secv1.SecretProviderClass{}, spcTestDefaultConf)
 		assert.False(t, ok)
 		require.NoError(t, err)
 	})
 
-	t.Run("incorrect ingress class", func(t *testing.T) {
-		ing := invalidURLIng.DeepCopy()
-		incorrect := "some-other-ingress-class"
-		ing.Spec.IngressClassName = &incorrect
-		ing.Annotations = map[string]string{"kubernetes.azure.com/tls-cert-keyvault-uri": "inv@lid URL"}
-
-		ok, err := i.buildSPC(ing, &secv1.SecretProviderClass{})
-		assert.False(t, ok)
-		require.NoError(t, err)
-	})
+	//t.Run("incorrect ingress class", func(t *testing.T) {
```

Comment:
why is this commented out?

Reply:
I meant to remove this test because it was a test case specifically for when the managed check logic was still in buildSPC. I moved that to the placeholderPodCleanCheck instead so it's not needed.